src/metabase/api/card.clj

```diff
@@ -608,20 +608,6 @@ saved later when it is ready."
     (validation/check-embedding-enabled)
     (api/check-superuser)))
 
-(defn- publish-card-update!
```

Comment:
there are no subscribers for card-unarchive and card-archive, also it has never been published before because archived and `(:card archive)` is identical when it's called this function.

so I've removed it